_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -971,12 +971,13 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
 
     if (attrs[ idx_map[VAConfigAttribEncROI] ].value != VA_ATTRIB_NOT_SUPPORTED) // VAConfigAttribEncROI
     {
-        VAConfigAttribValEncROI *VaEncROIValPtr = reinterpret_cast<VAConfigAttribValEncROI *>(&attrs[ idx_map[VAConfigAttribEncROI] ].value);
+        //!!!!!!WA: It need to fix when we will get valid caps.
```

Comment:
This is not right way to do! Please, file a bug against driver, file a bug against mediasdk with a link to the driver bug, provide a link here and close this PR.